### PR TITLE
Add java client

### PR DESCRIPTION
**Pull request by sujitb-nvidia**
Add an example showing how to use protoc compiler to generate a GRPC client library for java. The examples folder contains scala and java samples, the scala example is more comprehensive and shows how to use model and server API, apart from simple model inference

### CODE REVIEW

**Comment by deadeyegoodwin** [comment]
Can you change the description to say something like: "Add an example showing how to use protoc compiler to generate a GRPC client library for java. The example folder contains ... "

File: src/clients/java/README.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+
```
Comment (deadeyegoodwin) [inline src/clients/java/README.md:1]: Add title: "Example Java and Scale Client Using Generated GRPC API"

File: src/clients/java/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+## Prerequsites
+
+Maven 3.3+ and JDK 1.8+
+
+
```
Comment (deadeyegoodwin) [inline src/clients/java/README.md:6]: Remove the extra blank lines in this file

File: src/clients/java/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Generating java GRPC client stub
+
+Copy __*.proto__ files to Library src/main/proto
+```
+cd library
+cp ../../../core/*.proto src/main/proto/
+```
+
+__Note__ the *.proto files are copied from [triton-inference-server
+/server/src/core/](https://github.com/triton-inference-server/server/tree/master/src/core). The Libray should look as below.
```
Comment (deadeyegoodwin) [inline src/clients/java/README.md:17]: Libray -> "After copying the library directory should look ..."

File: src/clients/java/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+```
+mvn compile
+```
+Once compiled, one should notice the generated *.java files under __target__ folder
+
+<img src="images/grpc-stubs.png" width="400" />
+
+## Use the generated files in any project
+
+see the __examples__ project which has __scala__ and __java__ sample client.
```
Comment (deadeyegoodwin) [inline src/clients/java/README.md:31]: see -> See

File: src/clients/java/README.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+Maven 3.3+ and JDK 1.8+
+
+
+
+## Generating java GRPC client stub
+
+Copy __*.proto__ files to Library src/main/proto
+```
+cd library
```
Comment (deadeyegoodwin) [inline src/clients/java/README.md:12]: Put '$' in front for example cmdline:

$ cd library
$ cp ...

File: src/clients/java/README.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+```
+
+__Note__ the *.proto files are copied from [triton-inference-server
+/server/src/core/](https://github.com/triton-inference-server/server/tree/master/src/core). The Libray should look as below.
+
+
+<img src="images/proto-files.png" width="220" />
+
+```
+mvn compile
```
Comment (deadeyegoodwin) [inline src/clients/java/README.md:23]: $

File: src/clients/java/README.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+<img src="images/grpc-stubs.png" width="400" />
+
+## Use the generated files in any project
+
+see the __examples__ project which has __scala__ and __java__ sample client. 
+
+## Running java example client 
+
+```
+cd examples
```
Comment (deadeyegoodwin) [inline src/clients/java/README.md:36]: $
Combine these into one ``` region.

File: src/clients/java/examples/src/main/java/clients/SimpleJavaClient.java [hunk 8]
```diff
@@ -0,0 +1,122 @@
+package clients;
```
Comment (deadeyegoodwin) [inline src/clients/java/examples/src/main/java/clients/SimpleJavaClient.java:1]: Add copyright to this file. You can find it in any other source file

File: src/clients/java/examples/src/main/scala/clients/SimpleClient.scala [hunk 9]
```diff
@@ -0,0 +1,259 @@
+package clients
```
Comment (deadeyegoodwin) [inline src/clients/java/examples/src/main/scala/clients/SimpleClient.scala:1]: copyright

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]
Don't include the 3 generated files in the commit.

**Comment by deadeyegoodwin** [comment]
@sujitbiswas BTW, I'm not going to look at any updates until you let me know they are ready. You can do this by using the "re-request review" button next to my name in the upper right of the page.

**Comment by sujitb-nvidia** [comment]
> @sujitbiswas BTW, I'm not going to look at any updates until you let me know they are ready. You can do this by using the "re-request review" button next to my name in the upper right of the page.

Okay @deadeyegoodwin

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**